pmm-app/tests/qan_test.js

```diff
@@ -148,15 +150,83 @@ xScenario('Open the QAN Dashboard and check Details -> Tables', async (I, adminP
 
 Feature('To verify and test the QAN Dashboard query details tooltip');
 
-Before((I, loginPage) => {
-  I.amOnPage(loginPage.url);
-  loginPage.login('admin', 'admin');
+Before((I, qanPage) => {
+  I.Authorize();
+  I.amOnPage(qanPage.url);
 });
 
-xScenario('Open the QAN Dashboard and show tooltip', async (I, adminPage, qanPage) => {
+xScenario('Open the QAN Dashboard and show tooltip @new-qan', async (I, adminPage, qanPage) => {
   I.amOnPage(qanPage.url);
   qanPage.showTooltip(4, 1);
   I.wait(5);
   I.see('dasdadadad');
   // TODO: check changes in url
 });
+
+Scenario('Open the QAN Dashboard and check existence of filters @new-qan', async (I, qanPage) => {
+  qanPage.verifyFiltersSectionIsPresent();
+});
+
+Scenario('Open the QAN Dashboard and check work of button "reset all" @new-qan', async (I, qanPage) => {
+  qanPage.selectFilter(qanPage.fields.filterCheckboxSelector);
+  qanPage.resetAllFilters();
+  I.dontSeeCheckboxIsChecked(qanPage.fields.filterCheckboxSelector);
+});
+
+Scenario('Open the QAN Dashboard, add a column and make sure that this option not available anymore @new-qan', async (I, qanPage) => {
+  qanPage.addColumn('Bytes Sent');
```

Comment:
Bytes Sent can be stored as a variable on top of a test
add await for verifyColumnIsNotAvailable request